scripts/types.d.ts

```diff
@@ -0,0 +1,17 @@
+export type ComponentsEventHandlers = {
+  [name: string]: EventHandler[];
+};
+
+export interface EventHandler {
+  origin: string;
+  name: string;
+  description: string;
+  kind: "function";
+  type: string;
```

Comment:
maybe rename to `eventType` ?

Reply:
I don't want to change the API that codemodel currently exposes.